manifests/agent/packages.pp

```diff
@@ -0,0 +1,9 @@
+class classroom::agent::packages {
+  $packages = [
+    'openssl',         # used for generating password hashes for user resources
```

Comment:
I think it's safe to assume that. A lot of this can also be moved to the bootstrap once we've converged the courses to use puppetfactory, so we won't need to bother with caching packages.